<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QMainWindow" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>289</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVMonitor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resource.qrc">
    <normaloff>:/SVMonitor/images/logo.png</normaloff>:/SVMonitor/images/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>350</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Модули</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSlowPlay">
        <property name="text">
         <string>Тихий режим</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="treeWidgetExt" name="treeSignals">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>4</number>
       </property>
       <attribute name="headerDefaultSectionSize">
        <number>125</number>
       </attribute>
       <attribute name="headerShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Имя</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Тип</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Комментарий</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Группа</string>
        </property>
       </column>
      </widget>
      <widget class="QTextEdit" name="txtStatusMess"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>289</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionProgram"/>
   </widget>
   <addaction name="menuMain"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTrgPanel"/>
   <addaction name="actionEventOrder"/>
   <addaction name="actionPrint"/>
   <addaction name="actionExport"/>
   <addaction name="actionNewWin"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../resources/resource.qrc">
     <normaloff>:/SVMonitor/images/open.png</normaloff>:/SVMonitor/images/open.png</iconset>
   </property>
   <property name="text">
    <string/>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset>
     <normaloff>:/SVMonitor/resources/images/add.png</normaloff>:/SVMonitor/resources/images/add.png</iconset>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </action>
  <action name="actionSignals">
   <property name="icon">
    <iconset>
     <normaloff>:/SVMonitor/resources/images/icon.png</normaloff>:/SVMonitor/resources/images/icon.png</iconset>
   </property>
   <property name="text">
    <string>Сигналы</string>
   </property>
  </action>
  <action name="actionTrgPanel">
   <property name="icon">
    <iconset resource="../resources/resource.qrc">
     <normaloff>:/SVMonitor/images/flag-red.png</normaloff>:/SVMonitor/images/flag-red.png</iconset>
   </property>
   <property name="text">
    <string>Настройка уведомлений</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="actionEventOrder">
   <property name="icon">
    <iconset resource="../resources/resource.qrc">
     <normaloff>:/SVMonitor/images/icon.png</normaloff>:/SVMonitor/images/icon.png</iconset>
   </property>
   <property name="text">
    <string>Отчет событий</string>
   </property>
  </action>
  <action name="actionProgram">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionCheckUpdate">
   <property name="text">
    <string>Проверить обновления</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="../resources/resource.qrc">
     <normaloff>:/SVMonitor/images/print.png</normaloff>:/SVMonitor/images/print.png</iconset>
   </property>
   <property name="text">
    <string>Печать</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../resources/resource.qrc">
     <normaloff>:/SVMonitor/images/export.png</normaloff>:/SVMonitor/images/export.png</iconset>
   </property>
   <property name="text">
    <string>Экспорт</string>
   </property>
  </action>
  <action name="actionNewWin">
   <property name="icon">
    <iconset resource="../resources/resource.qrc">
     <normaloff>:/SVMonitor/images/add.png</normaloff>:/SVMonitor/images/add.png</iconset>
   </property>
   <property name="text">
    <string>Новое окно</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>treeWidgetExt</class>
   <extends>QTreeWidget</extends>
   <header>src/treeWidgetExt.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resource.qrc"/>
 </resources>
 <connections/>
</ui>
